### PR TITLE
SOF-1631: Key aliasing

### PR DESCRIPTION
**Pull request by KvelaGorrrrnio**
Introduces key aliasing, such that `Shift` is triggered by `ShiftLeft` and `ShiftRight`, `AnyEnter` is triggered by `Enter` and `NumpadEnter`, and so forth.

### CODE REVIEW

File: src/app/keyboard/services/tv2-key-alias.service.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { KeyAliasService } from '../abstractions/key-alias-service'
+
+export class Tv2KeyAliasService implements KeyAliasService {
+  public getAliasesForKey(key: string): string[] {
+    switch (key) {
+      case 'AnyEnter':
```
Comment (LindvedKrvang) [inline src/app/keyboard/services/tv2-key-alias.service.ts:6]: Should we have enums for this or is it overkill?

**Review by LindvedKrvang — APPROVED** [review]

File: src/app/keyboard/services/tv2-key-alias.service.ts (hunk 1 above)
Comment (KvelaGorrrrnio) [reply src/app/keyboard/services/tv2-key-alias.service.ts:6]: I thought about it, and I think it is a good idea, but currently we have the rule about enum values to be uppercased snakecase and that enum keys should be the same as the values, which makes it impossible for this.

Comment (LindvedKrvang) [reply src/app/keyboard/services/tv2-key-alias.service.ts:6]: We could make an exception for that rule?

Comment (KvelaGorrrrnio) [reply src/app/keyboard/services/tv2-key-alias.service.ts:6]: I'll do it, but I'll base it on the virtual keyboard branch.

**Review by andr9528 — APPROVED** [review]

**Pull request merged**